e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -413,6 +413,14 @@ def _get_climo_dataset(self, season: str) -> xr.Dataset:
         # ds = ds[[self.var, 'lat_bnds', 'lon_bnds']]
         ds = ds[[self.var] + keep_bnds]
```

Comment:
This line only keeps variable after derivation and bounds related data variables. It helps to remove excessive data for alleviating memory usage.  This change in dataset_xr.py may affect other sets.